app/models/container_group.rb

```diff
@@ -26,6 +26,8 @@ class ContainerGroup < ApplicationRecord
   belongs_to :old_container_project, :foreign_key => "old_container_project_id", :class_name => 'ContainerProject'
   belongs_to :container_build_pod
   has_many :container_volumes, :as => :parent, :dependent => :destroy
+  has_many :persistent_volume_claim, :through => :container_volumes
+  has_many :persistent_volumes, -> { where(:type=>'PersistentVolume') }, :through => :persistent_volume_claim, :source => :container_volumes
```

Comment:
is this our `:type` or is it the type over in `:persisted_volume_claim`?

Reply:
Do we even need the `where`?  I don't see any descendants of `PersistentVolume`